### PR TITLE
nobody related fixes

### PR DESCRIPTION
**Pull request by yuwata**
Follow-up for #7530.

(Sorry, I am afraid that I do not follow all the discussion in #7530...)

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -680,6 +680,11 @@ if id_result.returncode() == 0
                         '        Your build will result in an group table setup that is incompatible with the local system.')
         endif
 endif
+if nobody_user != nobody_group
+        message('WARNING:\n' +
+                '        The configured user name "@0@" and group name "@0@" of the nobody user/group are not equivalent.\n'.format(nobody_user, nobody_group) +
+                '        Please re-check that both "nobody-user" and "nobody-group" options are correctly set.')
```
Comment (keszybz) [inline meson.build]: Actually on debian the names are "nobody" and "nogroup", and we explicitly support that.

Comment (yuwata) [reply meson.build]: Oh, thank you. I did not know that. Indeed, nogroup is mentioned in UIDS-GIDS.md.

**Comment by yuwata** [comment]
@keszybz Thank you for the comment. I've force-pushed a new version. Now, warning is suppressed for the debian situation. Please take a look.

**Comment by poettering** [comment]
(actually i wouldn't mind warning even in the debian case, about the name mismatch... to push them gently towards just using nobody:nobody)

**Comment by keszybz** [comment]
`E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?`

**Pull request merged**